webapp/src/Controller/API/JudgehostController.php

```diff
@@ -1485,8 +1447,11 @@ public function getJudgeTasksAction(Request $request): array
         return [];
     }
 
-    /** @param JudgeTask[] $judgeTasks */
-    private function serializeJudgeTasks($judgeTasks, Judgehost $judgehost): array
+    /**
+     * @param JudgeTask[] $judgeTasks
```

Comment:
Is this needed?

Reply:
Yes since we can’t put this in a php type

Reply:
I would say that a variable named judgeTasks and which is an array should be considered a JudgeTask[] but PHPstorm might indeed not pick this up.